### ISSUE TITLE
Is tvOS supported ?

### ISSUE THREAD

**Issue by JanC** [orta/OROpenSubtitleDownloader#9]
Hi,
I see that there is a merged tvOS pull request https://github.com/orta/OROpenSubtitleDownloader/pull/8 but I'm getting some AFNetworking dependency errors on tvOS when using CocoaPods. Is this supposed to work?

```
[!] The platform of the target `MyKit` (tvOS 9.0) is not compatible with `AFNetworking (2.6.3)`, which does not support `tvos`.
```

cheers

**Comment by iGranDav** [comment]
@JanC my bad indeed I have a dependency on `AFNetworking` on my testing tvOS project which solves this bad dependency:
`pod 'AFNetworking', :git => 'https://github.com/AFNetworking/AFNetworking.git', :branch => '3_0_0_branch'`

The solution would be to remove the dependency on `AFNetworking` from the lib

**Comment by orta** [comment]
Yeah, I bet that can be converted to an NSURLSession request easily.

**Comment by JanC** [comment]
allright thanks

**Comment by iGranDav** [comment]
Reopening since only a workaround were provided, the issue is not solved yet.

**Comment by JanC** [comment]
For my case, I used a workaround with a  local podspec that has a dependency to AFNetworking 3 
```
s.dependency 'AFNetworking', '~> 3.0.0-beta.2'
```

**Comment by iGranDav** [comment]
@JanC you could do that indeed but since `OROpenSubtitleDownloader` does not specify a version for AFNetworking simply adding `pod 'AFNetworking', :git => 'https://github.com/AFNetworking/AFNetworking.git', :branch => '3_0_0_branch'` in your project podfile solves also the issue.
